src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -162,4 +162,28 @@ public function provideServersSetting()
             );
         }
     }
+
+    /**
+     * @dataProvider provideDSNWithOptions
+     */
+    public function testDSNWithOptions($dsn, array $options)
+    {
+        $client = MemcachedAdapter::createConnection($dsn);
+
+        foreach ($options as $option => $expect) {
+            $this->assertSame($expect, $client->getOption($option));
+        }
+    }
+
+    public function provideDSNWithOptions()
```

Comment:
should be `provideDsnWithOptions`
the provider should do a class_exists check on Memcached and yield nothing when not, in order to allow skipping the tests when Memcached is not installed